fastlane/lanes/build.rb

```diff
@@ -0,0 +1,427 @@
+  #
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  #
+  # @called_by CI
+  # @calls build_and_upload_internal
+  #
+  desc 'Builds and uploads for distribution to App Center'
+  lane :build_and_upload_app_center do |options|
+    ios_build_prechecks(
+      skip_confirm: options[:skip_confirm],
+      internal: true,
+      external: true
+    )
+
+    ios_build_preflight
+
+    build_and_upload_internal(
+      skip_prechecks: true,
+      skip_confirm: options[:skip_confirm]
+    )
+  end
+
+  # Builds the WordPress app for an Installable Build ("WordPress Alpha" scheme), and uploads it to AppCenter
+  #
+  # @called_by CI
+  #
+  desc 'Builds and uploads an installable build'
+  lane :build_and_upload_installable_build do
+    sentry_check_cli_installed
+
+    alpha_code_signing
+
+    # Get the current build version, and update it if needed
+    version_config_path = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+    versions = Xcodeproj::Config.new(File.new(version_config_path)).to_hash
+    build_number = generate_installable_build_number
+    UI.message("Updating build version to #{build_number}")
+    versions['VERSION_LONG'] = build_number
+    new_config = Xcodeproj::Config.new(versions)
+    new_config.save_as(Pathname.new(version_config_path))
+
+    gym(
+      scheme: 'WordPress Alpha',
+      workspace: WORKSPACE_PATH,
+      export_method: 'enterprise',
+      clean: true,
+      output_directory: BUILD_PRODUCTS_PATH,
+      output_name: 'WordPress Alpha',
+      derived_data_path: DERIVED_DATA_PATH,
+      export_team_id: ENV['INT_EXPORT_TEAM_ID'],
+      export_options: { method: 'enterprise' }
+    )
+
+    appcenter_upload(
+      api_token: get_required_env('APPCENTER_API_TOKEN'),
+      owner_name: 'automattic',
+      owner_type: 'organization',
+      app_name: 'WPiOS-One-Offs',
+      file: lane_context[SharedValues::IPA_OUTPUT_PATH],
+      dsym: lane_context[SharedValues::DSYM_OUTPUT_PATH],
+      destinations: 'All-users-of-WPiOS-One-Offs',
+      notify_testers: false
+    )
+
+    sentry_upload_dsym(
+      auth_token: get_required_env('SENTRY_AUTH_TOKEN'),
+      org_slug: 'a8c',
+      project_slug: 'wordpress-ios',
+      dsym_path: lane_context[SharedValues::DSYM_OUTPUT_PATH]
+    )
+
+    return if ENV['BUILDKITE_PULL_REQUEST'].nil?
+
+    download_url = Actions.lane_context[SharedValues::APPCENTER_DOWNLOAD_LINK]
+    UI.message("Successfully built and uploaded installable build here: #{download_url}")
+    install_url = 'https://install.appcenter.ms/orgs/automattic/apps/WPiOS-One-Offs/'
+
+    comment_body = "You can test the <strong>WordPress</strong> changes on this Pull Request by downloading it from AppCenter <a href='#{install_url}'>here</a> with build number: <code>#{build_number}</code>. IPA is available <a href='#{download_url}'>here</a>. If you need access to this, you can ask a maintainer to add you."
```

Comment:
```suggestion
    comment_body = "You can test the <strong>WordPress</strong> changes on this Pull Request by downloading it from App Center <a href='#{install_url}'>here</a> with build number: <code>#{build_number}</code>. IPA is available <a href='#{download_url}'>here</a>. If you need access to this, you can ask a maintainer to add you."
```

Reply:
Actually, while we're at it, I think this might be clearer:

```suggestion
    comment_body = "You can test the changes in <strong>WordPress</strong> from this Pull Request by downloading it from AppCenter <a href='#{install_url}'>here</a> with build number: <code>#{build_number}</code>. IPA is available <a href='#{download_url}'>here</a>. If you need access to this, you can ask a maintainer to add you."
```